synfig-studio/src/gui/canvasview.cpp

```diff
@@ -2992,6 +3012,16 @@ CanvasView::on_waypoint_clicked_canvasview(ValueDesc value_desc,
 		item->show();
 		waypoint_menu->append(*item);
 
+		item = manage(new Gtk::MenuItem(_("_Copy")));
+		item->set_use_underline(true);
+		item->signal_activate().connect(
+		sigc::bind(sigc::mem_fun(*this, &CanvasView::copy_waypoints), waypoint_set));
+		item->show();
+		waypoint_menu->append(*item);
+
+		//is a paste here necessary?
```

Comment:
Yes, it is, if there is something to paste. :)

Reply:
hmm ok makes sense, if someone needs to paste at the same place of another waypoint.

Reply:
done